src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static Task[] taskList = new Task[100];
+    private static int taskNum = 0;
+
+    private static void listTasks() {
```

Comment:
You could state the function description for the listTasks as a comment if you want to be super clear.